mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTextView.js

```diff
@@ -58,6 +58,7 @@ class ShowArtifactTextView extends Component {
         marginTop: '0',
         width: '100%',
         height: '100%',
+        padding: '5px',
```

Comment:
This is unintentional, right?

Reply:
Yes it is, I wasnt aware of any other files in this commit.

Reply:
Could you revert this change?

Reply:
Maybe I should've reverted the whole commit and not just the change